recipe/meta.yaml

```diff
@@ -29,6 +29,7 @@ about:
 
 extra:
   recipe-maintainers:
+    # N.B., this is the master list of users who (aside from conda- danielnachun
```

Comment:
This looks like a bot error. Instead of adding him to the list, it concatenated his username to the end of the comment. I'm guessing the bot isn't expecting comments in this section.